src/psyclone/transformations.py

```diff
@@ -2768,13 +2769,13 @@ def validate(self, sched, options=None):
             # parameters (issue 323) we have to bypass this validation and
             # provide them manually for the OpenCL kernels to compile.
             continue
```

Comment:
Still `continue` because the OpenCL backend would still fail even with the evaluated  `use` statements.

We will delete the continue when we have the UseToArguments transformations that  (I hope) we can now implement because PSyclone has all the information about the symbols to create the argument signature. But I will do this in a follow-up PR.